packages/react-components/src/TextArea.tsx

```diff
@@ -0,0 +1,144 @@
+// Copyright 2017-2020 @polkadot/react-components authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import React, { TextareaHTMLAttributes, useCallback } from 'react';
+import styled from 'styled-components';
+
+import { Labelled } from '.';
+import { ThemeProps } from './types';
+
+interface Props {
+  children?: React.ReactNode;
+  className?: string;
+  help?: React.ReactNode;
+  isError?: boolean;
+  isReadOnly?: boolean;
+  label?: React.ReactNode;
+  onChange?: (arg: string) => void;
+  seed?: string;
+  withLabel?: boolean;
+}
+
+interface TextInputProps extends TextareaHTMLAttributes<any>{
+  withError?: boolean;
+}
+
+const TextInput = ({ withError, ...textAreaProps }: TextInputProps) => <textarea
+  {...textAreaProps}
+  className={[textAreaProps.className, withError ? 'ui-textArea-withError' : ''].join(' ')}
+/>;
+
+const TextAreaInput = styled(TextInput)(({ theme }: ThemeProps) => `
+  background: ${theme.bgInput};
+  border-radius: 4px;
```

Comment:
```suggestion
  border-radius: 0.25rem;
```

Just aligns with what we have everywhere.